tests/db.rs

```diff
@@ -45,3 +46,42 @@ pub fn clear(connection: &Connection) {
 
     connection.execute("TRUNCATE TABLE sentence;", &[]).unwrap();
 }
+
+/// Inserts one sentence into the table 'sentence'
+///
+/// # Arguments:
+///
+/// `connection` - The PostgreSQL connection object
+/// `uuid` - The sentence UUID v4
+/// `content` - The sentence itself
+/// `iso639_3` - The sentence language
+///
+/// NOTE: allow `unused_must_use` because we don't need to get the result of execute(),
+/// allow `dead_code` as this function is only used by the tests
```

Comment:
Do you have any idea why I get a "warning dead code" only for this method and not for the previous one ?

Reply:
By the way, I simplified the problem and I reported an issue to the Rust project here (https://github.com/rust-lang/rust/issues/46379). Let's simply move forward.